interface/wx/webview.h

```diff
@@ -460,11 +460,33 @@ class wxWebView : public wxControl
     virtual void Reload(wxWebViewReloadFlags flags = wxWEBVIEW_RELOAD_DEFAULT) = 0;
 
     /**
-        Runs the given javascript code.
-        @note When using wxWEBVIEW_BACKEND_IE you must wait for the current
+        Runs the given javascript code, returning the result if there is any.
+        It returns strings, integers, floating point numbers, booleans and objects (as JSON).
+        @note When using wxWEBVIEW_WEBKIT (OSX), there are two limits:
+              1) JavaScript allocations greater than 10MB.
+              2) JavaScript that takes longer than 10 seconds to execute.
+
+              When using wxWEBVIEW_BACKEND_IE you must wait for the current
               page to finish loading before calling RunScript().
-    */
-    virtual void RunScript(const wxString& javascript) = 0;
+
+              When using wxWEBVIEW_BACKEND_IE, JSON is not available in Quirks or 
+              IE6/7 standards mode, which is unfortunately the default one for the embedded browser control, see
+              https://docs.microsoft.com/en-us/scripting/javascript/reference/json-object-javascript#requirements
+              and see here how to make a program run use "modern" modes
+              https://msdn.microsoft.com/en-us/library/ee330730(v=vs.85)#browser_emulation
+              Here is an example to get this working on webview.exe:
+              @code
+                  wxRegKey key(wxRegKey::HKCU, _T("SOFTWARE\\Microsoft\\Internet Explorer\\Main\\FeatureControl\\FEATURE_BROWSER_EMULATION"));
```

Comment:
Wouldn't be then also expected to have a method that returns the installed IE version so it is known which modes can actually work? Perhaps also a method which returns the IE mode the application is  run in?